release-notes/README.md

```diff
@@ -42,6 +42,10 @@ This SDK is compatible with Android Core 1.8.2 and above.
 
 ## June 8, 2021
 
+### iOS Campaign Standard 3.0.0
+
+* Initial release to support [Adobe Campaign Standard workflows](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/master/using-mobile-extensions/adobe-campaign-standard) for Adobe Experience Platform Mobile SDKs for iOS in Swift. This extension library is [available as an open source project on Github](https://github.com/adobe/aepsdk-campaign-ios/).
```

Comment:
nit: let's remove the closing slash character from https://github.com/adobe/aepsdk-campaign-ios